bin/xbps-query/defs.h

```diff
@@ -63,7 +63,7 @@ int	list_pkgs_pkgdb(struct xbps_handle *);
 int	repo_list(struct xbps_handle *);
 
 /* from search.c */
-int	search(struct xbps_handle *, bool, const char *, const char *, bool);
+int	search(struct xbps_handle *, bool, const char *, const char *, bool, bool);
```

Comment:
Adding third bool parameter looks like good time to introduce enum search_flags.